tests/manager/manager-ctor_error-003.phpt

```diff
@@ -0,0 +1,266 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): invalid types in URI options arrays
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+/* Note: generic boolean options (e.g. "ssl") are not tested because the driver
+ * uses bson_iter_as_bool() to cast the value to a boolean for assignment. This
+ * may be changed in PHPC-990.
```

Comment:
I prefer not having an issue mentioned in here, as we might likely forget to remove it when PHPC-990 gets fixed—unless this test starts failing when we address PHPC-990.